services/app-api/src/postgres/contractAndRates/proto_to_db_JoinTable.ts

```diff
@@ -0,0 +1,54 @@
+import type { PrismaClient } from '@prisma/client'
+
+async function migrateAssociations(
+    client: PrismaClient
+): Promise<void | Error> {
+    try {
+        // Retrieve all submitted ContractRevisions.
```

Comment:
From RSP: Look at where migrating relationships for drafts is happening if at all